### PR TITLE
Nhdd mappings

### PR DESCRIPTION
**Pull request by Justus446**

### CODE REVIEW

**Comment by karimkkanji** [comment]
Resolve the conflict then restore the keys as they are required when the key is found when sent by echis in the payload of the screening object.

**Comment by karimkkanji** [comment]
Once done, this should close #110

File: src/utils/referral.js [hunk 1]
```diff
@@ -79,36 +81,188 @@ const echisNHDDValuesCoding = {
     code: `29541`,
     display: `Lack of sleep`
   },
-  feeling_of_worthlessness: {
-    system: SNOMED_CT_URL,
-    code: `247892001`,
-    display: `Feeling of worthlessness`
-  },
+  // feeling_of_worthlessness: {
```
Comment (freddieptf) [inline src/utils/referral.js:84]: This is required for [PNC Mother](https://github.com/moh-kenya/config-echis-2.0/blob/main/forms/app/postnatal_care_service.xlsx), i believe it's part of mental danger signs. Some of the mappings commented out might actually be needed

**Review by freddieptf — CHANGES_REQUESTED** [review]
Great effort @Justus446, we just need to confirm that we have the mappings in this [document](https://docs.google.com/document/d/147EpDeTUrh3u5CokxQRLcMAh30jlVRxc0ZH-ro11ivo/edit#heading=h.q4vzhc7zk1fe). Let's make sure we didn't comment out mappings that we need as well

**Comment by Justus446** [comment]
As agreed upon last Friday.Some tickets have mappings that need an expert (There no one to one mapping of the eCHIS symptoms to NHDD codes).Also there's previous work done by the design team that does not match to the codes in NHDD. [Link to the doc](https://docs.google.com/spreadsheets/d/1cowY7uIR7bMCHTVQV8YxcRRolAYTy-Zd/edit#gid=1081183559)
@kitsao , Kindly reassign the ticket.

**Comment by freddieptf** [comment]
Closing this, worked on in #119 #115

**Pull request closed without merging**